### PR TITLE
Some small improvements here and there

### PR DESCRIPTION
**Pull request by bokic**

### CODE REVIEW

**Comment by dridk** [comment]
Whaouu!! Thanks a lot !

File: analysis/analysisrunner.cpp [hunk 1]
```diff
@@ -227,11 +240,11 @@ QString AnalysisRunner::humanFileSize() const
 {
     int unit;
     const char *units [] = {" Bytes", " kB", " MB", " GB"};
-    quint64 size = fileSize(); // or whatever
+    qint64 size = fileSize(); // or whatever
```
Comment (natir) [inline analysis/analysisrunner.cpp:243]: A file can be have a negative size ?

File: analysis/analysisrunner.cpp [hunk 2]
```diff
@@ -227,11 +240,11 @@ QString AnalysisRunner::humanFileSize() const
...
 
     for (unit=-1; (++unit<3) && (size>1023); size/=1024);
 
-    return QString::number(size, 'f', 1) + QString(units[unit]);
+    return QString::number(size, 'f', 1) + QString::fromLatin1(units[unit]);
```
Comment (natir) [inline analysis/analysisrunner.cpp:247]: What justified this change ?

**Review by natir — COMMENTED** [review]
Very Big thanks for this modification. Just two point need to be clarify.

File: analysis/analysisrunner.cpp (hunk 1 above)
Comment (natir) [reply analysis/analysisrunner.cpp:243]: Ok I check the Qt documentation QFileInfo::size return a qint64. It's strange, but silent cast isn't a good idea

**Comment by bokic** [comment]
dridk please close this pull request because I did a regression in one commit.

I will make new pull request soon.

File: analysis/analysisrunner.cpp (hunk 2 above)
Comment (bokic) [reply analysis/analysisrunner.cpp:247]: QString(const char *str) constructor creates QString from UTF-8,

compared to

QString::fromLatin1(const char *str, int size = -1) that creates QString from Latin-1 charset.

http://doc.qt.io/qt-5/qstring.html#QString-7
http://doc.qt.io/qt-5/qstring.html#fromLatin1

File: analysis/analysisrunner.cpp (hunk 1 above)
Comment (bokic) [reply analysis/analysisrunner.cpp:243]: Windows is using signed filesize too.

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383713(v=vs.85).aspx

QuadPart
A signed 64-bit integer.

**Pull request closed without merging**